src/credentials/examples/DeviceAttestationCredsExample.cpp

```diff
@@ -52,7 +52,11 @@ class ExampleDACProvider : public DeviceAttestationCredentialsProvider
 
 CHIP_ERROR ExampleDACProvider::GetDeviceAttestationCert(MutableByteSpan & out_dac_buffer)
 {
+#if 0x8000 <= CHIP_DEVICE_CONFIG_DEVICE_PRODUCT_ID && CHIP_DEVICE_CONFIG_DEVICE_PRODUCT_ID <= 0x801F
```

Comment:
instead of hardcoding the 0x8000 here, can we define the `CHIP_DEVICE_CONFIG_DEVICE_PRODUCT_ID_MIN/MAX` or something in the ExampleDACs.h file? 

It'd be nicer to have the control of the range specified close to where they're defined otherwise if we add additional product identifiers here you'd need to know to look at this spot too otherwise it won't work and you'd be wondering why

Reply:
Actually, wouldn't it be better to simply add a GN switch not to compile `ExampleDACProvider` if you use a non-default product ID? This class is only for test purposes so it shouldn't be used if you have your own DAC. Btw, why is the linker error occuring? Is `ExampleDACProvider` instantiated in your application? Wouldn't removing `ExampleDACProvider` from the application be enough to resolve it?

Reply:
Done, in this PR: https://github.com/project-chip/connectedhomeip/pull/25953

Closing this one.